evalml/data_checks/null_data_check.py

```diff
@@ -15,21 +15,38 @@ class NullDataCheck(DataCheck):
     Args:
         pct_null_col_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
             that column will be considered highly-null. Defaults to 0.95.
+        pct_moderately_null_col_threshold(float): If the percentage of NaN values in an input feature exceeds this amount
+             but is less than the percentage specified in pct_null_col_threshold, that column will be considered
+             moderately-null. Defaults to 0.20.
         pct_null_row_threshold(float): If the percentage of NaN values in an input row exceeds this amount,
             that row will be considered highly-null. Defaults to 0.95.
     """
 
-    def __init__(self, pct_null_col_threshold=0.95, pct_null_row_threshold=0.95):
+    def __init__(
+        self,
+        pct_null_col_threshold=0.95,
```

Comment:
Do we want to change the name of this to reflect that this is "more null" than "moderately null"?  Like "heavily null" or something.  I dunno, I'm not a thesaurus.

Reply:
The only reason I chose not to was to prevent this from becoming a breaking change!